lib/matplotlib/_mathtext.py

```diff
@@ -2599,3 +2604,23 @@ def auto_delim(self, s, loc, toks):
             # if "mid" in toks ... can be removed when requiring pyparsing 3.
             toks["mid"].asList() if "mid" in toks else [],
             toks["right"])
+
+    def substack(self, s, loc, toks):
+        state = self.get_state()
+        thickness = state.get_current_underline_thickness()
+        vlist = []
+
+        max_width = sorted(toks[1:], key= lambda c: c.width)[-1].width
```

Comment:
Maybe this could be written using `max`, which also takes a `key` argument.
```
max_width = max(toks[1:], key=lambda c: c.width)
```
Something like that?